tidb-cloud/learning-path.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Learning Path
+hide_title: true
+hide_sidebar: true
+---
+
+<!--
+注意frontmatter里的内容，hide了右边栏和文章的title。
```

Comment:
Would you please translate **all** the Chinese text in this file to English? Let's use English in this repo. Thanks!